include/swift/AST/DiagnosticsClangImporter.def

```diff
@@ -228,7 +228,18 @@ ERROR(foreign_reference_types_invalid_retain,none,
 ERROR(foreign_reference_types_invalid_release,none,
       "specified release function '%0' is invalid; release function must have exactly "
       "one argument of type '%1'", (StringRef, StringRef))
-
+ERROR(foreign_reference_types_invalid_retain_return_type, none,
+      "specified retain function '%0' is invalid; retain function must have "
+      "void return type",
```

Comment:
No strong feelings here, but I wonder if we usually put any reference to source code in singe quotes, so I wonder if it should be 'void'. That being said, this is not a direct quote from the code where the warning is triggered, so not entirely sure.

Reply:
Agreed, we should probably have void in single quotes.